src/serac/numerics/functional/detail/hexahedron_H1.inl

```diff
@@ -73,5 +96,385 @@ struct finite_element<Geometry::Hexahedron, H1<p, c> > {
     return dN;
     // clang-format on
   }
+

```

Comment:
I know it's boiler plate for every element type, but it would be good to write some prose on the meaning of `B` and `G`

Reply:
I wonder why doxygen isn't producing warnings about these functions